components/nimbus/Cargo.toml

```diff
@@ -35,6 +35,7 @@ once_cell = "1"
 uniffi = { version = "^0.17", optional = true }
 chrono = { version = "0.4", features = ["serde"]}
 unicode-segmentation = "1.8.0"
+glean = "42.3.2"
```

Comment:
hmmm I don't feel too strongly about this, but what do you think about removing this and adding it back when we can actually consume the dependency (once we have it in iOS). That will keep the scope of this PR on the `metrics.yaml` generation from build scripts and will save us the unused dependency

Regardless I don't mind keeping it, I checked the binary sizes as well and given our megazords, it's a very small increase

Reply:
I'd be perfectly fine with that, given that it is very easily added back.